packages/core/src/components/Pagination.story.tsx

```diff
@@ -61,4 +61,46 @@ storiesOf('Core/Pagination', module)
       page={2}
       pageCount={3}
     />
+  ))
+  .add('Align arrows at the start', () => (
+    <Pagination
+      startAlign
+      hasNext
+      hasPrev
+      showBookends
```

Comment:
small nit - we do try to alphabetize props. so `hasNext`, `hasPrev`, `showBookends`, `startAlign`. same comment in other examples

Reply:
Agh, you know I usually do that too! Must have missed these, will fix.